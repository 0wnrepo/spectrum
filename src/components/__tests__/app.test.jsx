import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from '../app';

describe('<App />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <App />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
