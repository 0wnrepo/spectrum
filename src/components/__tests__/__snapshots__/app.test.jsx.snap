// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<div
  className="pusher"
>
  <Connect(TransactionSigningOverlay) />
  <HashRouter>
    <MenuSystem
      className="content"
      fixed="top"
      marginTop="5em"
      renderLastItem={[Function]}
      secondary={false}
      tabs={
        Array [
          Object {
            "component": [Function],
            "exact": true,
            "icon": "key",
            "name": "Keystores",
            "path": "/",
          },
          Object {
            "component": [Function],
            "icon": "wrench",
            "name": "Config",
            "path": "/config",
          },
          Object {
            "component": [Function],
            "icon": "code",
            "name": "Dapplets",
            "path": "/dapplets",
          },
        ]
      }
      usingRouter={true}
    />
  </HashRouter>
  <Footer />
</div>
`;
